packages/core/src/Resource/Repository.js

```diff
@@ -37,11 +37,11 @@ export default function Repository(connector, resourceType, identifier) {
     },
     /**
      * @param {String|Number} id
-     *
+     * @param {ParameterBag} parameterBag
      * @returns {Promise<Resource>}
      */
-    findById(id) {
-      return connector.fetchOne(this, id, {}).then(
+    findById(id, parameterBag) {
+      return connector.fetchOne(this, id, (typeof parameterBag === 'undefined') ? {} : parameterBag).then(
```

Comment:
Might be better to do a default value for the parameter and don't do any checks here at all